pkg/kongstate/util.go

```diff
@@ -82,10 +82,17 @@ func getPlugin(s store.Storer, namespace, name string) (kong.Plugin, error) {
 func kongPluginFromK8SClusterPlugin(
```

Comment:
`kongPluginFromK8SClusterPlugin` and `kongClusterPluginFromK8SClusterPlugin` need to be tested, at least to the extent of changes made here.

Reply:
8112614 adds new tests for these specific to kongstate. Existing parser tests _should_ have covered these already; not sure why CodeCov thinks they don't. `parser.Build()` calls that include `Kong(Cluster)Plugins` in their store (e.g. https://github.com/Kong/kubernetes-ingress-controller/blob/v1.1.1/internal/ingress/controller/parser/parser_test.go#L142-L179) have to run these functions, and we should have coverage of most of their content through tests there anyway.

That aside, it's better to have tests independent of the parser anyway, so whatever.